core/modules/css/CssError.h

```diff
@@ -33,9 +33,12 @@
 #define LSST_QSERV_CSSERROR_H
 
 // System headers
-#include <map>
 #include <stdexcept>
 #include <string>
+#include <boost/lexical_cast.hpp>
```

Comment:
we consider boost headers "third party headers", so add appropriate comment above, and switch to " "